### ISSUE TITLE
Crop the image without the UI

### ISSUE THREAD

**Issue by kaoz70** [CrackerakiUA/ui-cropper#31]
Hello!

Is there a way to crop the image automatically without showing the crop window? I cant find anything in the docs or source. 

This would be usefull for example when you crop the images automatically on upload and have an edit button to show a UI for cropping.

I hope I make sense.

**Comment by CrackerakiUA** [comment]
add ng-hide=true

**Comment by kaoz70** [comment]
Yeah, that was the first thing I tried, but I'm relying on the cropject attribute to get the currently cropped image and when I hide the directive I get other data:

Hidden:
`{
  "canvasSize": {
    "w": 300,
    "h": 225
  },
  "areaCoords": {
    "x": 0,
    "y": 37.5,
    "w": 300,
    "h": 150
  },
  "cropWidth": 300,
  "cropHeight": 150,
  "cropTop": 37.5,
  "cropLeft": 0,
  "cropImageWidth": 1280,
  "cropImageHeight": 640,
  "cropImageTop": 160,
  "cropImageLeft": 0
}`

Not hidden:

`{
  "canvasSize": {
    "w": 500,
    "h": 375
  },
  "areaCoords": {
    "x": 0,
    "y": 62.5,
    "w": 500,
    "h": 250
  },
  "cropWidth": 500,
  "cropHeight": 250,
  "cropTop": 62.5,
  "cropLeft": 0,
  "cropImageWidth": 1280,
  "cropImageHeight": 640,
  "cropImageTop": 160,
  "cropImageLeft": 0
}`

They are diferent, probably because they rely on the canvas size.

**Comment by CrackerakiUA** [comment]
then don't hide it, move it somewhere where it will not be visible.

**Comment by kaoz70** [comment]
OK that worked, not the most elegant solution, but its fine for me. Thanks!

**Comment by CrackerakiUA** [comment]
Yes, i have made such thing before, not elegant way for sure.

**Comment by CrackerakiUA** [comment]
This is probably quick modification. Need to add option which will decide if canvas will be real or not.
